modern/src/map/Map.js

```diff
@@ -35,7 +35,27 @@ const updateReadyValue = value => {
   readyListeners.forEach(listener => listener(value));
 };
 
+const onIdle = function () {
```

Comment:
I would move it to `setAllowedZoomLevels`. It's really only relevant there.

Reply:
I would, but it is an event listener function and when I want to remove the event listener I need to provide the same function to `map.off()` function. If I move it to `setAllowedZoomLevels`, a new function is created every time and `map.off()` can't remove the event listener.